### PR TITLE
PFM-TASK-3291 - Adjust @cplace/asc to use default registry

### PR DESCRIPTION
**Pull request by slaven3kopic**
Resolves [PFM-TASK-3291](https://base.cplace.io/pages/gj90e58qe5ab2blxhp6kpyllq/PFM-TASK-3291-Adjust-cplace-asc-to-use-default-registry)

**Checklist:**
- [x] Version updated (if applicable)
- [x] Code formatted

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -118,17 +118,17 @@ jobs:
             - name: Link
               run: npm link
 
-            - name: combineJavascript main
+            - name: cplace-asc main
               working-directory: ./main
               timeout-minutes: 20
-              run: ./gradlew combineJavascript
+              run: cplace-asc --verbose
```
Comment (stefancplace) [inline .github/workflows/continuous-integration.yml:124]: why is gradle no longer involved ?

**Comment by stefancplace** [comment]
on local execution the following happens to .npmrc:
```
@fortawesome:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/

@cplace-next:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm-test/
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de

@cplace-3rdparty-modified:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/
//cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:_auth=auth=
//cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:always-auth=true
//cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:email=stefan.stadler@collaboration-factory.de

@cplace-next:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
```

is changed to:

```
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
//cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de

```

Some lines are duplicated but the config is valid

**Review by stefancplace — APPROVED** [review]
some minor remarks, altogether LGTM

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (Kaltenbach) [reply .github/workflows/continuous-integration.yml:124]: I'd guess that's due to: we try to separate the gradle and npm stuff as good as possible. so IIRC the gradle plugin just kicks of the asc compilation to make sure the assets are part of the software.zip but does not combine the JavaScripts any longer or has other 'duties' considering frontend compilation.

**Comment by slaven3kopic** [comment]
> on local execution the following happens to .npmrc:
> 
> ```
> @fortawesome:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/
> 
> @cplace-next:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm-test/
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
> 
> @cplace-3rdparty-modified:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/
> //cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:_auth=auth=
> //cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:always-auth=true
> //cplace.jfrog.io/artifactory/api/npm/cplace-assets-npm/:email=stefan.stadler@collaboration-factory.de
> 
> @cplace-next:registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
> ```
> 
> is changed to:
> 
> ```
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
> registry=https://cplace.jfrog.io/artifactory/api/npm/cplace-npm/
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:_auth=auth=
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:always-auth=true
> //cplace.jfrog.io/artifactory/api/npm/cplace-npm/:email=stefan.stadler@collaboration-factory.de
> ```
> 
> Some lines are duplicated but the config is valid

I will take a look 👍

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (slaven3kopic) [reply .github/workflows/continuous-integration.yml:124]: Correct. Vladimir added this change to cplace-asc few releases back

**Pull request merged**